src/modules/keyboardmanager/common/KeyboardManagerState.h

```diff
@@ -50,6 +52,10 @@ class KeyboardManagerState
     // Stores the UI element which is to be updated based on the shortcut entered
     StackPanel currentShortcutUI;
     std::mutex currentShortcutUI_mutex;
+    
+    // Stores the current configuration name.
+    std::wstring currentConfig;

```

Comment:
Use a `default` value. This is will fail for running on first time because noconfig would be present.

Reply:
I think we just call the initial one default.json, and so that will also be the default activeconfiguration. Right now it creates a file called `.json` the first time, but doesn't actually fail. On the second load it opens `config-1.json` though.